### DISCUSSION TITLE
同一个sql，多个数据库执行

### DISCUSSION THREAD

**Discussion by KleinLee93** [hhyo/Archery#1664] [Q&A]
同一个sql语句需要在三个数据库实例中执行，目前只能通过提交3个sql上线工单解决。
可以有方法，只提交一个工单，同时选中多个数据库实例吗
![archery](https://user-images.githubusercontent.com/108920828/177935910-119f209b-34c7-4b9f-ac58-0a70ebd8bde9.png)

**Comment by nick2wang** [comment]
目前不支持同时提交到多个实例，即使将来要实现，也不应该合并为一个工单，不同实例的执行结果独立且可能存在差异；当前只能麻烦多点几次“上线其他实例”了:smile:

**Reply by KleinLee93** [reply]
感谢解答
